src/services/Admin/index.js

```diff
@@ -5,31 +5,14 @@ import { fetchGqlData } from '../shared';
 import request from 'request';
 
 const SETTINGS_ENDPOINT = 'settings';
-
-const trustedTwitterFragment = `fragment FortisTrustedTwitterAcctView on TrustedTwitterAccountCollection {
-                            accounts {
-                                    RowKey
-                                    acctUrl
-                            }
-                        }`;
-
-const fbPageAnalyticsFragment = `fragment FortisAdminSettingsView on FacebookPageAnalyticsCollection {
-                        analytics {
-                            Name,
-                            Count,
-                            LastUpdated
-                        }
-                      }`;
-
-const fbPageFragment = `fragment FortisDashboardView on FacebookPageCollection {
-                        runTime
-                        pages {
-                            RowKey
-                            pageUrl
-                        }
-                      }`;
+const MESSAGES_ENDPOINT = 'messages';
 
 export const SERVICES = {
+  restartPipeline(callback) {
+    const query = `${AdminMutations.restartPipeline}`;
+    fetchGqlData(MESSAGES_ENDPOINT, { query }, callback);
```

Comment:
Might as well have `const variables = {}` here for consistency with the methods below.